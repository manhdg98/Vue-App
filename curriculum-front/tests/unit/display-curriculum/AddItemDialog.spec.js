import { mount } from '@vue/test-utils'
import AddItemDialog from '@/components/display-curriculum/AddItemDialog.vue'

describe('AddItemDialog.vue', () => {
  test('if snapshot is correct', () => {
    const wrapper = mount(AddItemDialog, {
      stubs: {
        VDialog: {
          template: '<div></div>'
        }
      },
      propsData: {
        saveItem: () => {},
        toggleDialog: () => {},
        sectionIndex: 0,
        dialog: {
          type: '',
          show: false,
          name: '',
          url: '',
          isEditing: false,
          itemIndex: null,
          sectionIndex: null
        }
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
