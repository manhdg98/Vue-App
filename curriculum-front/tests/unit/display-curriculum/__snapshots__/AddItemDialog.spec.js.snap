// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItemDialog.vue if snapshot is correct 1`] = `<div width="500" persistent=""></div>`;
